frontend/src/lib/components/Link.js

```diff
@@ -7,6 +7,10 @@ export function Link({ to, preventClick = false, ...props }) {
             href={to || '#'}
             {...props}
             onClick={(event) => {
+                if (event.ctrlKey) {
+                    window.open(to, '_blank')
```

Comment:
Could you do `return window.open(to, '_blank')`? Now if you open it in a new tab it'll also load the page in the current tab

Reply:
done :smile: